### PR TITLE
Show broken metadata

### PR DESCRIPTION
**Pull request by fosskers**
## TODO
- [x] Alter `gt/metadata` endpoint to serve raw, broken JSON if possible
- [x] Add "Raw Metadata" button to client

## Motivation
Layers may have broken metadata that no longer conform to the `TileLayerMetadata[SpatialKey]` JSON format. To debug layers like these, we'd like a way to easily view their raw metadata as JSON through the viewer client.

### CODE REVIEW

File: client/src/main/scala/geotrellis/admin/client/routes/GeotrellisAdminViewer.scala [hunk 1]
```diff
@@ -39,7 +40,7 @@ object GeotrellisAdminViewer {
             "Metadata JSON"
           ),
           (if (state.showJSON) {
-            <.code("This is code")
+            props().displayM.rawMetadata.render(j => <.pre(j))
```
Comment (fosskers) [inline client/src/main/scala/geotrellis/admin/client/routes/GeotrellisAdminViewer.scala]: @moradology Should this be moved into its own React component?

**Comment by fosskers** [comment]
This view is hidden by default. By pressing the `Metadata JSON` button, we see:

json-button

File: server/src/main/scala/geotrellis/admin/server/AdminActor.scala [hunk 2]
```diff
@@ -170,11 +188,11 @@ trait AdminRoutes extends HttpService with CORSSupport {
       val layerId = LayerId(layer, zoom)
 
       pathPrefix("grid")(tmsGrid(layerId, key)) ~
-      pathPrefix("type")(tmsType(layerId, key)) ~
-      pathPrefix("breaks")(tmsBreaks(layerId, key)) ~
-      pathPrefix("histo")(tmsHisto(layerId, key)) ~
-      pathPrefix("stats")(tmsStats(layerId, key)) ~
-      pathEnd(serveTile(layerId, key))
+        pathPrefix("type")(tmsType(layerId, key)) ~
```
Comment (moradology) [inline server/src/main/scala/geotrellis/admin/server/AdminActor.scala]: Is this indentation intentional? In general, I think it's a good idea, but I find these path specifications most readable when indentation suggests going a step deeper in context - these just seem to be sibling routes

Comment (fosskers) [inline server/src/main/scala/geotrellis/admin/server/AdminActor.scala]: I agree. That's the ensime reformatter being a bit zealous.

File: client/src/main/scala/geotrellis/admin/client/routes/GeotrellisAdminViewer.scala (hunk 1 above)
Comment (moradology) [inline client/src/main/scala/geotrellis/admin/client/routes/GeotrellisAdminViewer.scala]: I feel like this is probably a fine way to handle it until more functionality is required

**Comment by moradology** [comment]
Looks good - +1 upon the minor formatting change

**Comment by fosskers** [comment]
Yeah I'll grab that right away.

**Comment by fosskers** [comment]
...Actually since I have another branch which is based off this one, I'll just fix it there. Should make for a cleaner git log.

**Comment by moradology** [comment]
Sounds good

**Pull request merged**